.eslintrc.yaml

```diff
@@ -13,3 +13,4 @@ parserOptions:
 
 rules:
   react/jsx-uses-vars: [2]
+  react/jsx-uses-react: [2]
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md

without this `source/js/components/loading-indicator/loading-indicator.jsx` will throw a linting error